### PR TITLE
Add ILSpy.app v7.0-rc1

### PR DESCRIPTION
**Pull request by bbenzikry**
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.
- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by bbenzikry** [comment]
Pending https://github.com/icsharpcode/AvaloniaILSpy/issues/64

**Comment by bbenzikry** [comment]
@miccal Can you please explain why you are updating this to a failing state?

**Comment by miccal** [comment]
> @miccal Can you please explain why you are updating this to a failing state?
> I've disabled maintainer edits for now

I was just checking if the `livecheck` block was still necessary (which, it appears, is still necessary).

**Comment by bbenzikry** [comment]
> > @miccal Can you please explain why you are updating this to a failing state?
> > I've disabled maintainer edits for now
> 
> I was just checking if the `livecheck` block was still necessary (which, it appears, is still necessary).

Why not check this locally with ``brew audit``?

**Comment by bbenzikry** [comment]
@miccal would love for this to be merged

File: Casks/ilspy.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask "ilspy" do
+  version "7.0-rc1"
```
Comment (SMillerDev) [inline Casks/ilspy.rb:2]: Isn't there a stable version?
https://docs.brew.sh/Acceptable-Casks#stable-versions

Comment (bbenzikry) [reply Casks/ilspy.rb:2]: No, there isn't - previous versions were also only released as pre releases and the repository was a fork. Both those issues were fixed with the developers help on the mentioned issue. 

https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version :smile:

**Comment by miccal** [comment]
@bbenzikry please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by bbenzikry** [comment]
> @bbenzikry please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

Done

**Review by miccal — APPROVED** [review]

**Comment by christophwille** [comment]
For documentation purposes: can be found at https://formulae.brew.sh/cask/ilspy#default

**Pull request merged**